talent/hiring_process.md

```diff
@@ -0,0 +1,77 @@
+- The Talent Manager posts the job description through Lever on the source{d} website.
+
+### Sourcing
+
+- Lead engineers are responsible to identity people to be contacted and other sourcing channels such as niche job boards. 
+
+- At any given moment the Talent Manager is able to drop in and work on finding more developers.
+
+- Developer lists/sourcing channels should continuously be defined and refreshed after communication between the Talent Manager and the leads. 
+
+- The Talent Manager is responsible to work with the Developer Community team in order to gain people through community actions (Twitter/Blog etc.). 
+
+### First Contact
+
+- Any developer should be contacted by the lead engineers or under their name at least (after together defining the outgoing message). In case the Talent Manager is contacting them, it should be done from the lead's email acccount. It's a motivating factor for the developers to be contacted by the lead engineers. 
+
+### Screening
+
+- If the developer is an applicant, they should be accepted/rejected by the lead engineers during the screening phase. 
+
+- If the developer is approached by us from a specific source (known from meetups, having worked together in the past,  etc.), they should move forward to the next step as it means that the leads have already shown interest.
+
+### First Skype/Zoom/Phone Interview
+
+- A quick Skype/Phone/Zoom call should be available to every developer who is interested in the roles*. It is a general discussion to know more about the background and experience of the developer. And to share as much as possible from our side about the team & company. 
+
+\* There are noteable exceptions to this, such as higher volume junior roles where we directly ask the developer to do the coding challenge before a call. 
+
+- Lead engineers are responsible to make the calls as it's a motivating factor for the developers. Leads will also be able to assess early in the process if the interview process should continue or not. 
+
+### Coding Challenges 
+
+- Every developer who passes the first Skype interview, should complete the coding challege. 
+
+- In case of the Machine Learning team, there is an additional ML task necessary to be solved.
+
+- Coding Challenges are being sent to the developers by the Talent Manager and the results are being reviewed by the lead engineers. 
+
+### On-site Interview
+ 
+- This is the last step in which it's  decided if the developer will receive an offer or not. It might last a full day or be split across two half-days, depending on the availability of the developer. 
+
+- The interviews are the following: 
+
+* Kickoff with the CTO, QA: to know the developer better
+* Design interview: to see how the developer would design a solution to a given technical problem working with a several of our engineers. In case of a developer being interviewed for the Infrastructure team, we have to make sure that this interview covers the specified needs for the team.
```

Comment:
"with several of our engineers"

Reply:
Why only the Infrastructure?
I'd say each time has specific needs that should be assured before accepting a candidate.

Reply:
I think Applications, ML and Data Retrieval have similar problems to give to the candidate as there is intersection between them, comparing to the Infrastructure. But writing in a more general way, making sure that the needs of should be assured for every team covers the issue that had been faced by Infrastructure team.